runbooks, Add CnaoNotReady alert (#19)
kubevirt/monitoring: docs/runbooks/CnaoNotReady.md

# CnaoNotReady

## Meaning

> Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.

Reviewer: I am not sure we need to add the CNAO is, looks like the alarm is happening when "additional networking components" are deployed.

They don't explain what kubevirt is here https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeVirtComponentExceedsRequestedCPU.md#meaning

Maybe we can talk about CNAO CR

```
This alert fires when CNAO CR is not ready, which means that one of the deployed components is not ready.

Author: > They don't explain what kubevirt is here https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeVirtComponentExceedsRequestedCPU.md#meaning

but they do explain it on an HCO alert [KubevirtHyperconvergedClusterOperatorUSModification](https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubevirtHyperconvergedClusterOperatorUSModification.md) :) 

But I like your line about the CNAO CR, will change that, but let's continue to discuss if the CNAO intro thing is needed..

Author: As discussed on #18 , will keep this part.

This alert fires when one or more of CNAO deployed components is not ready.

## Impact

> Without CNAO components deployed and ready, the user might lack certain network components
> that are provided by this component.

Reviewer: Let's talk about the impact on use case for kubevirt.

```suggestion
Without CNAO components deployed and ready, not all the kubevirt networking scenarios can be executed
```

## Diagnosis

- Check for messages on the networkaddonsconfig CR status:
	```
	kubectl get networkaddonsconfig -o custom-columns="":.status.conditions[*].message

> 	```

Reviewer: Maybe extracting the deployment/daemonSet have to be here and also readin the logs.

Author: fixed.

## Mitigation

> Identify which component is not ready, check the logs for the root cause and fix it.

Reviewer: Identify is the diagnosis, mitigation is something like "kill the component pods", or "kill CNAO pods", or re apply CR.

# CnaoNotReady

## Meaning

Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.

> This alert fires when CNAO CR is not ready, which means that one of the deployed components is not ready.

Reviewer: Let's add the name of the CR kind NetworkAddonsConfig

Author: https://github.com/kubevirt/monitoring/pull/21
